multisig/proposals/opsepolia/vip-005/index.ts

```diff
@@ -0,0 +1,50 @@
+import { NETWORK_ADDRESSES } from "src/networkAddresses";
+import { makeProposal } from "src/utils";
+
+const { opsepolia } = NETWORK_ADDRESSES;
+
+export const ACM = "0x1652E12C8ABE2f0D84466F0fc1fA4286491B3BC1";
+export const PSR = "0x0F021c29283c47DF8237741dD5a0aA22952aFc88";
+export const NATIVE_TOKEN_GATEWAY_CORE_POOL = "0xfE0AAFd5A4fEB9f6DEC36B0Cb8A972bD0B5149f7";
```

Comment:
According to https://github.com/VenusProtocol/isolated-pools/pull/435/

```suggestion
export const NATIVE_TOKEN_GATEWAY_CORE_POOL = "0x521f59b2670bcc70961FB2edD4F62c3f3B1E7f6b";
```

Reply:
Yes, both addresses are right. I just lost the NTG deployment file while merging this branch with main so needed to deploy the NTG one more time. `0x521f59b2670bcc70961FB2edD4F62c3f3B1E7f6b ` latest one which is mentioned here :https://github.com/VenusProtocol/isolated-pools/pull/435
Anyways, I have updated the address here as well, https://github.com/VenusProtocol/isolated-pools/pull/435